README.md

```diff
@@ -23,6 +23,66 @@ A recalculations run requires that reprocess has been performed a priori, and it
 ./run_sorting_hat_test_data.sh EXPS_DIR
 ```
 
-The experiments path contains one or more directories with Atlas accession names E-* (e.g. `E-MTAB-5577`), having at least configuration files in xml format after curation process. Completed processing by iRAP Single Lib (ISL) is necessary before new experiment processing.
+The experiments path contains one or more directories with Atlas accession names E-* (e.g. `E-MTAB-5577`), having at least configuration files in xml format after curation process.
 
 Optionally, worflow execution can be tailored to specific accessions or species by defining these variables in the sorting-hat script.
+
+### New experiment processing and re-processing
+
+Completed processing by iRAP Single Lib (ISL) is necessary before new experiment processing. It will run all rules avilable for the experiment type.
+
+### Recalculations
+
+This is necessary for the Ensembl Update (E! Update) part of a Data Release. E! Update brings annotations from Biomart, E! Mysql databases and ftp sites for all the relevant organisms in Expressiona Atlas, and leaves them in a format that can be consumed for the decoration process and the web applications. Ensembl Update validators (for Biomart attributes and GTF URL validations) are performed before running recalculations.
```

Comment:
Ensembl Update validators also looks for existence of MySQL database

Reply:
[ecd4644](https://github.com/ebi-gene-expression-group/bulk-recalculations/pull/85/commits/ecd4644ebe53e6bb354d40505410bcc211cb11f4)